zerver/views/auth.py

```diff
@@ -977,7 +977,7 @@ def json_fetch_api_key(
         if not authenticate(
             request=request, username=user_profile.delivery_email, password=password, realm=realm
         ):
-            raise JsonableError(_("Your username or password is incorrect."))
+            raise JsonableError(_("Entered password is incorrect."))
```

Comment:
I think we can just say "Password is incorrect."

Reply:
sure, making it 'Password is incorrect'.